client-side/src/stories/DateTime/DateInputs.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import DateInputs from './DateInputs';
```

Comment:
the datetime isn't part of your task, please take it from existing files in main branch